test/auth/sasl-imap-smtp-auth.txt

```diff
@@ -0,0 +1,5 @@
+EHLO mail
```

Comment:
That's a option. perhaps LDAP users may have an idea on that specific point.
But I'm currently refactoring tests and I can say that's not obvious when we look at the different combinations with these variables.